source/quantity.js

```diff
@@ -53,17 +53,16 @@ Quantity = Space.domain.ValueObject.extend('Quantity', {
     return new Quantity(this.value - toSubstract.value);
   },
 
-  delta: function(other) {
+  delta(other) {
     return this.value - this._getCleanValue(other);
   },
 
-  _getCleanValue: function(other) {
+  _getCleanValue(other) {
     return (other instanceof Quantity) ? other.value : other;
   }
 
 });
 
-// Register EJSON type
 Quantity.type('Quantity');
```

Comment:
Type registration is done by Space.messaging.Serializable so this should not be needed